Polyhedron/demo/Polyhedron/Scene.cpp

```diff
@@ -1158,6 +1158,19 @@ void Scene::printPrimitiveIds(CGAL::Three::Viewer_interface* viewer)
       item->printPrimitiveIds(viewer);
   }
 }
+void Scene::updatePrimitiveIds(CGAL::Three::Viewer_interface* viewer, CGAL::Three::Scene_item* it)
+{
+  if(it)
+  {
+    //Only call printPrimitiveIds if the item is a Scene_print_interface_item
+    Scene_print_interface_item* item= dynamic_cast<Scene_print_interface_item*>(it);
+    if(item)
+    {
+      item->printPrimitiveIds(viewer);
+       item->printPrimitiveIds(viewer);
```

Comment:
... Would you mind writing a comment, there, to explain that fact? :-)

And also fix the indentation.